docs/static/spec/openapi/starter.yaml

```
@@ -0,0 +1,252 @@
+                    mem: 
+                      heap_used_percent: 14
+                      heap_committed_in_bytes: 309866496
+                      heap_max_in_bytes: 1037959168
+                      heap_used_in_bytes: 151686096
+                      non_heap_used_in_bytes: 122486176
+                      non_heap_committed_in_bytes: 133222400
+                      pools: 
+                        survivor:
+                          peak_used_in_bytes: 8912896
+                          used_in_bytes: 288776
+                          peak_max_in_bytes: 35782656
+                          max_in_bytes: 35782656
+                          committed_in_bytes: 8912896
+
+  /_node/hot_threads:
+    get:
+      summary: Get hot threads
+      description: >
+        Get information about current hot threads for Logstash.
+        A hot thread is a Java thread that has high CPU usage and takes longer than normal to execute. 
+      operationId: nodeHot_threads
+      parameters:
+        - name: threads
+          in: query
+          schema:
+            type: integer
+          description: >
+            The number of hot threads to return. The default is 10. 
+        - name: stacktrace_size
+          in: query
+          schema:
+            type: integer
+          description: >
+            The depth of the stack trace to report for each thread. The default is 50. 
+        - name: ignore_idle_threads
+          in: query
+          schema:
+            type: boolean
+          description: >
+            If true, does not return idle threads. The default is `true`. 
+        - name: pretty
+          in: query
+          schema:
+            type: boolean
+          description: >
+            If you append `?pretty=true` to the request, the JSON returned will be pretty formatted. Use it for debugging only!
+        - name: human
+          in: query
+          schema:
+            type: boolean
+          description: >
+            If you append `?human=true` to the request, the JSON returned will be in a human-readable format.
+      responses:
+        '200':
+          description: Indicates a successful call
+          content:
+            application/json:
+              examples:
+                nodeHotThreadsExample1:
+                  hot_threads:
+                    - time: 2025-01-06T18:25:28-07:00
+                      busiest_threads: 3
+                      threads: 
+                        name: Ruby-0-Thread-7
+                        percent_of_cpu_time: 0.0
+                        state: timed_waiting
+                        path: /path/to/logstash-8.17.0/vendor/bundle/jruby/1.9/gems/puma-2.16.0-java/lib/puma/thread_pool.rb:187
+                        traces: java.lang.Object.wait(Native Method)", "org.jruby.RubyThread.sleep(RubyThread.java:1002)", "org.jruby.RubyKernel.sleep(RubyKernel.java:803)
+
+  /_health_report:
+    get:
+      summary: Get health status
+      description: >
```

```suggestion
      description: |
```
Use literal scalar instead of folding to preserve newlines.